cloudformation.yaml

```
@@ -975,6 +1021,35 @@ Resources:
       Threshold: 2
       TreatMissingData: notBreaching
 
+  FeastGooglePubsub5xxErrors:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled:
+        !FindInMap [StageVariables, !Ref Stage, AlarmActionsEnabled]
+      AlarmActions:
+        - Ref: AlarmTopic
+      OKActions:
+        - Ref: AlarmTopic
+      AlarmName: !Sub mobile-purchases-${Stage}-feast-google-pubsub-check-errors
+      AlarmDescription: A HTTP request to the Feast Google pubsub endpoint resulted in a 5XX error.
+      Dimensions:
+        - Name: ApiName
+          Value: !Sub ${App}-${Stage}
+        - Name: Method
+          Value: POST
+        - Name: Resource
+          Value: /feast/google/pubsub
+        - Name: Stage
+          Value: !Ref Stage
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 1
+      MetricName: 5XXError
+      Namespace: AWS/ApiGateway
+      Period: 300
+      Statistic: Sum
+      Threshold: 2
```

if the description is indended as "A Http request.... (failed)" then it should be 1.
Also for "1" error, the alarm comes through up to 4 minutes quicker if it's a 1 minute period
```suggestion
      Period: 60
      Statistic: Sum
      Threshold: 1
```

---

Yes, I'm wondering what we actually want from this alarm. We have the same alarm rule for all the pubsub endpoints, and we'd probable want to keep them the same whichever way we end up going.

---

Will address in a separate PR, trying to improve the alarm for all 4 endpoints